### PR TITLE
Create AccountSettings screen.

### PR DESCRIPTION
**Pull request by MiqayelChaloyan**
### Created account settings screen

  *I created the ability to add and change name, email and address.* <br>
  *I have created an option to add a card.*  <br>
 
photo_2023-07-06_17-26-17
photo_2023-07-06_17-26-38
photo_2023-07-06_17-26-22

### CODE REVIEW

File: components/CustomInput/CustomTextInput.jsx [hunk 1]
```diff
@@ -16,20 +16,21 @@ const CustomTextInput = ({
...
 
   return (
-    <>
+    <View style={styles.container}>
       <TextInput
         name={name}
         placeholder={placeholder}
         placeholderTextColor={theme.colors.cool_gray}
-        style={styles.input}
+        style={style || styles.input}
```
Comment (Varuj2002) [inline components/CustomInput/CustomTextInput.jsx:33]: add to default props
defaultProps
style: styles.input

File: components/DataActiveView/index.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    }
+
+    return (
+        <View style={styles.container}>
+            <Text style={selectAppropriateStyle('value', isCreditCard)}>
+                {value}
+            </Text>
+            <View style={styles.checked}>
+                {isCreditCard ? (
+                    <TouchableOpacity style={styles.checkedRemove} onPress={handleDeleteCard}>
```
Comment (Varuj2002) [inline components/DataActiveView/index.js:34]: <TouchableOpacity style={styles.checkedRemove} onPress={handleDeleteCard}>
                        <Text style={styles.removeCard}>✖️</Text>
</TouchableOpacity>
  this code is repeated, move to separate component

File: components/BottomSheet/index.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+const BottomSheet = ({children, onClose}) => {
+  const [sheetHeight, setSheetHeight] = useState(Dimensions.get('window').height * 2);
+
+  const handleClose = useCallback(() => {
+    setSheetHeight(0);
+    onClose();
+    }, [onClose]);
+
+  return (
+    <TouchableOpacity
```
Comment (Varuj2002) [inline components/BottomSheet/index.js:15]: Why use TouchableOpacity

File: components/CustomInput/CustomTextInput.jsx (hunk 1 above)
Comment (MiqayelChaloyan) [reply components/CustomInput/CustomTextInput.jsx:33]: ok, done

File: components/BottomSheet/index.js (hunk 3 above)
Comment (MiqayelChaloyan) [reply components/BottomSheet/index.js:15]: it is unnecessary here, i will change it.

**Pull request closed without merging**